### PR TITLE
Few fixes for Weave Scope

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

File: weavescope/manifests/scope-probe-ds.yaml [hunk 1]
```diff
@@ -3,31 +3,36 @@ kind: DaemonSet
...
+        - "--no-app"
+        - "--probe.docker"
+        - "true"
+        - "--probe.docker.bridge"
+        - "docker0" # change this in case the Docker bridge is named `cbr0` in your system
+        - "--probe.kubernetes"
+        - "true"
+        - "weave-scope-app"
+        hostPID: true
+        hostIPC: true
```
Comment (2opremio) [inline weavescope/manifests/scope-probe-ds.yaml]: I don't think the probe runs in the hosts IPC

Comment (errordeveloper) [inline weavescope/manifests/scope-probe-ds.yaml]: yeah, I've copied that from elsewhere - thought it wont hurt...

Comment (2opremio) [inline weavescope/manifests/scope-probe-ds.yaml]: Well, let's try to reflect how the weaveworks/scope Docker container does it

**Comment by errordeveloper** [comment]
see #4

**Pull request closed without merging**